indigo_app/static/javascript/indigo/views/library.js

```diff
@@ -369,13 +369,39 @@
         work.n_docs = work_docs.length;
         work.n_docs_singular = work_docs.length == 1;
 
+        if (work.drafts_v_published.n_published) {
+          work.n_published = work.drafts_v_published.n_published
```

Comment:
This is simpler and works when `work.drafts_v_published.n_published` is false, null or 0.

```js
work.n_published = work.drafts_v_published.n_published || 0;
```